### PR TITLE
PUBDEV-7875 - replace System.out with Log

### PR DESCRIPTION
**Pull request by valenad1**
https://h2oai.atlassian.net/browse/PUBDEV-7875

### CODE REVIEW

**Review by michalkurka — CHANGES_REQUESTED** [review]
Please see inline comments

File: h2o-algos/src/main/java/hex/glm/GLM.java [hunk 1]
```diff
@@ -49,6 +50,9 @@
  * Generalized linear model implementation.
  */
 public class GLM extends ModelBuilder<GLMModel,GLMParameters,GLMOutput> {
+
+  private static final Logger LOG = Logger.getLogger(GLM.class);
```
Comment (michalkurka) [inline h2o-algos/src/main/java/hex/glm/GLM.java:54]: This class is already using `water.util.Log` to handle logging

We should stick with one - I think in this PR you probably don't want to redo everything. cc: @honzasterba

**Review by honzasterba — APPROVED** [review]
👍

File: h2o-algos/src/main/java/hex/glm/GLM.java (hunk 1 above)
Comment (valenad1) [reply h2o-algos/src/main/java/hex/glm/GLM.java]: Log4j removed and water.util.Log is used

**Comment by honzasterba** [comment]
it seems you based your branch on master and are trying to merge it to zermelo

**Comment by valenad1** [comment]
> 
> 
> it seems you based your branch on master and are trying to merge it to zermelo

Exactly, It is weird that rebase this branch on rel-zermelo did not work as I expected and shows those 58 commit. Now I reset hard on rel-zermelo and cherry pick those two commits.

**Comment by honzasterba** [comment]
rebase will not work in this case, beacause it will take all the commits that are not common and rebase on the new branch, hence duplicating the commits, you can do `git rebase -i new-branch` to see what you are bringing over and keep only what you really want to rebase

**Review by michalkurka — APPROVED** [review]
Thank you for making the changes @valenad1

**Pull request merged**